src/olympia/versions/migrations/0025_auto_20220614_1653.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2.13 on 2022-06-14 16:53
+
+from django.db import migrations
+
+from olympia import amo
+from olympia.addons.tasks import index_addons
+from olympia.versions import compare
+
+
+def update_dictionary_compat(apps, schema_editor):
+    Version = apps.get_model('versions', 'Version')
+    ApplicationsVersions = apps.get_model('versions', 'ApplicationsVersions')
+    AppVersion = apps.get_model('applications', 'AppVersion')
+    Addon = apps.get_model('addons', 'Addon')
+
+    # Find the versions without correct compatibility
+    version_ids = list(
+        Version.unfiltered.filter(addon__type=amo.ADDON_DICT, deleted=False).exclude(
+            addon__status=amo.STATUS_DELETED,
+            apps__min__version=amo.DEFAULT_WEBEXT_DICT_MIN_VERSION_FIREFOX,
+            apps__max__version=amo.DEFAULT_WEBEXT_MAX_VERSION,
+        )
+    )
+    if version_ids:
+        webext_dict_min, _ = AppVersion.objects.get_or_create(
+            application=amo.FIREFOX.id,
+            version=amo.DEFAULT_WEBEXT_DICT_MIN_VERSION_FIREFOX,
+            version_int=compare.version_int(amo.DEFAULT_WEBEXT_DICT_MIN_VERSION_FIREFOX),
```

Comment:
We need the `AppVersion` for `61.0` to definitely exist to be able to fk to it - it _should_ already on dev/stage/prod but possibly won't locally.  

`version_int` is a numeric representation of the version string for comparison, and usually it's generated from `version` automatically in `AppVersion.save`. Using `get_or_create` on the `objects` manager that the fake models that are used in migrations (the `Addon` that `apps.get_model` returns) doesn't call the `save` method but it's required so we're generating it here instead.

Reply:
wrapping it in `if version_ids` is to work around test CI - all the migrations are run before the test suit starts so it would otherwise end up creating two `AppVersion`s in the database each time, and some tests rely on there being no (or a known set of) existing `AppVersion`s.